Gems/Atom/Feature/Common/Code/Source/ReflectionProbe/ReflectionProbeFeatureProcessor.cpp

```diff
@@ -283,6 +283,18 @@ namespace AZ
             probe->ShowVisualization(showVisualization);
         }
 
+        void ReflectionProbeFeatureProcessor::SetRenderExposure(const ReflectionProbeHandle& probe, float renderExposure)
+        {
+            AZ_Assert(probe.get(), "SetRenderExposure called with an invalid handle");
+            probe->SetRenderExposure(renderExposure);
+        }
+
+        void ReflectionProbeFeatureProcessor::SetBakeExposure(const ReflectionProbeHandle& probe, float bakeExposure)
+        {
+            AZ_Assert(probe.get(), "SetBakeExposure called with an invalid handle");
+            probe->SetBakeExposure(bakeExposure);
+        }
```

Comment:
This seems strange to have a wrapper function like this. Why not just have the client code call ReflectionProbe::SetRenderExposure directly? 

I see for other functions like SetProbeOuterExtents we have some extra steps being taken, so maybe you are just following that pattern. But if it's necessary for client code to call through ReflectionProbeFeatureProcessor then I think the API should enforce this in some way. MeshFeatureProcessor accomplishes this by making everything in MeshDataInstance private, with MeshFeatureProcessor being a friend. Have you considered doing that kind of thing here?

Reply:
I prefer a functional interface at the boundary, with the ReflectionProbe as an internal renderer object and the client interfacing strictly with the feature processor.

Reply:
Sure, that's fine. In that case, should the accessors inside ReflectionProbe be private, so the client code is forced to work through the feature processor?

(I recognize this is outside the scope of the current PR).